src/Sulu/Bundle/SecurityBundle/Tests/Unit/Domain/Event/UserCreatedEventTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\Tests\Unit\Domain\Event;
+
+use PHPUnit\Framework\TestCase;
+use Sulu\Bundle\SecurityBundle\Domain\Event\UserCreatedEvent;
+use Sulu\Component\Security\Authentication\UserInterface;
+
+class UserCreatedEventTest extends TestCase
```

Comment:
i think @luca-rath did not add such test cases in his PRs. most tests are probably quite basic, so i dont have a strong opinion here. you dont need to add them if you dont want to 😅

Reply:
😇 😂

Reply:
I like the tests because they ensure, that the correct `resourceKey` and `eventType` is used and they are implemented pretty fast